test/integration/image-component/default/test/index.test.js

```diff
@@ -494,6 +494,15 @@ function runTests(mode) {
       )
     })
 
+    it('should show error when not numeric string width or height', async () => {
+      const browser = await webdriver(appPort, '/invalid-weight-or-height')
```

Comment:
```suggestion
      const browser = await webdriver(appPort, '/invalid-width-or-height')
```

After renaming the file, update here too 👍